lego/apps/lending/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+from lego.apps.files.models import FileField
+from lego.apps.lending.permissions import LendableObjectPermissionHandler
+from lego.apps.permissions.constants import VIEW
+from lego.apps.permissions.models import ObjectPermissionsModel
+from lego.apps.users.models import User
+from lego.utils.models import BasisModel
+
+
+class LendableObject(BasisModel, ObjectPermissionsModel):
+    title = models.CharField(max_length=128, null=False, blank=False)
+    description = models.TextField(null=False, blank=True)
+    image = FileField(related_name="lendable_object_images")
```

Comment:
You're using `image` and `images`. Is this plural, or is it just possible to have one image?

Reply:
Only one image!